packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -673,6 +673,10 @@ interface HandleCloudflareChallengeOptions {
     isChallengeCallback?: (page: Page) => Promise<boolean>;
     /** Allows overriding the detection of Cloudflare "blocked page". */
     isBlockedCallback?: (page: Page) => Promise<boolean>;
+    /** Allows overriding how the checkbox click position is calculated. */
+    clickPositionCallback?: (page: Page) => Promise<{ x: number; y: number } | null>;
+    /** Optional delay (in milliseconds) before the first click attempt on the challenge checkbox. */
+    waitPeriod?: number;
```

Comment:
```suggestion
    waitPeriodMillis?: number;
```

or maybe we should use seconds here, it feels weird to have one time based option in millis and the other in seconds. also, since we wait for one second on each iteration, what about making that time configurable instead of introducing another sleep option in the same loop?